package-lock.json

```diff
@@ -9019,14 +9019,15 @@
       }
     },
     "node_modules/follow-redirects": {
-      "version": "1.15.3",
+      "version": "1.15.5",
```

Comment:
Is this update intentional?

Reply:
this was changed after running the npm ci command before committing 
not something I did manually, but since there is a new file that was added to the projects, I think it correct to update the version number

Reply:
@AlexSCorey  is that OK with you?
something else need to be done?

Reply:
I loaded the file from main branch 
so a lot of changes now

Reply:
done!
the file is back to origin